### ISSUE TITLE
New config seems to be incompatible with MySQL

### ISSUE THREAD

**Issue by TiltedToast** [drizzle-team/drizzle-kit-mirror#86]
I followed the "guide" to update my drizzle config file for 0.19 but it seems to not be very happy with something so it just fails every time.

Here's the new config

```ts
import "dotenv/config";
import { Config } from "drizzle-kit";

export default {
    schema: "./src/db/models/*.ts",
    breakpoints: true,
    dbCredentials: {
        connectionString: process.env.PLANETSCALE_URL,
    },
} satisfies Config;
```

I did try to specify the driver but that changes nothing, neither does playing around with the credentials part of the config. 

Here's the error it throws 

```bash
> drizzle-kit push:mysql --config ./src/db/drizzle.config.ts

drizzle-kit: v0.19.1
drizzle-orm: v0.27.0

Reading config file 'E:\coding-projects\hifumi-js\src\db\drizzle.config.ts'
node:internal/errors:490
    ErrorCaptureStackTrace(err);
    ^

TypeError [ERR_INVALID_RETURN_PROPERTY_VALUE]: Expected a url string to be returned for the "url" from the "drizzle-kit/loader.mjs 'resolve'" function but got instance of String.
    at new NodeError (node:internal/errors:399:5)
    at ESMLoader.resolve (node:internal/modules/esm/loader:882:15)
    at async ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:7) {
  code: 'ERR_INVALID_RETURN_PROPERTY_VALUE'
}

Node.js v18.16.0
```

I did some more testing and the same thing happens when I try to do anything that involves the config above.

**Comment by wfl-junior** [comment]
I am getting the same error when trying to use it with better-sqlite3

**Comment by rodbs** [comment]
Same error with sqlite

**Comment by pavlovtech** [comment]
Getting the same error on Windows 11. I'm using PlanetScale serverless driver

**Comment by phenomenal-hardy** [comment]
having the exact same error, on windows 11 and postgreSQL

**Comment by AndriiSherman** [comment]
Fixed in `drizzle-kit@0.19.2`
